### DISCUSSION TITLE
Come trasformare un ordine in ricevuta o fattura

### DISCUSSION THREAD

**Discussion by bellamatte** [fattureincloud/api#82] [Q&A] [answered]
Ho creato con successo un ordine tramite API e ora vorrei automatizzare la parte di trasformazione in fattura o ricevuta.

Come posso fare?

Grazie.

**Comment by emajo** [comment]
Ciao @bellamatte attualmente le chiamate per la trasformazione dei documenti non sono ancora documentate, ci stiamo lavorando e rilasceremo il prima possibile.

Emanuele.

**Reply by bellamatte** [reply]
Ciao @emajo , 
ci sono novità su questo argomento?

Riesci a dirmi più o meno quanto tempo ci vorrà?

Grazie,
Matteo

**Reply by emajo** [reply]
Ciao @bellamatte la chiamata per trasformare un documento puoi trovarla [qui](https://developers.fattureincloud.it/api-reference#get-/c/-company_id-/issued_documents/transform), nei prossimi giorni verranno rilasciate le SDKs con i nuovi metodi e successivamente anche una guida per l'utilizzo.

Grazie,
Emanuele.

**Comment by valmoz** [comment, marked as answer]
Buongiorno @bellamatte 
Abbiamo rilasciato la guida per la trasformazione di un documento e le SDK relative.

[Join & Trasform Issued Documents](https://developers.fattureincloud.it/docs/guides/invoice-join-transform)

Grazie
Mauro
